packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -178,6 +187,7 @@
                   </template>
                   <template if:true={canShowMore}>
                     <button
+                      data-testid="facet-values__show-more"
```

Comment:
do we still want this here since we changed the selector in quanticFacet pageObject to :

`this.facet.getByRole('button', {
      name: /Show more values for the .* facet/i,
    });`

Reply:
oh nvm you use it for unit tests I see